src/sensors.cpp

```diff
@@ -339,6 +362,16 @@ bool Sensors::update_imu(void)
   }
 }
 
+void Sensors::update_battery_average(float new_voltage)
+{
+  battery_voltage_history_[battery_monitor_filter_pointer_++]=new_voltage;
+  if(battery_monitor_filter_pointer_ == BATTERY_MONITOR_MOVING_AVERAGE_COUNT)
+    battery_monitor_filter_pointer_ = 0;
+  float voltage_sum{0};
+  for(float voltage: battery_voltage_history_)
+    voltage_sum += voltage;
```

Comment:
It should be possible to implement this without having to sum over the whole array every time, right?

Like
```cpp
data_.battery_voltage += (battery_voltage_history[newest_index] - battery_voltage_history_[oldest_index]) / BATTERY_MONITOR_MOVING_AVERAGE_COUNT;
```
Is that worth doing to save a few cycles? Or does it just not matter?